packages/components/table/src/table-column/render-helper.ts

```diff
@@ -57,10 +64,17 @@ function useRender<T>(
     const source = cellForced[type] || {}
     Object.keys(source).forEach((prop) => {
       const value = source[prop]
-      if (value !== undefined) {
-        column[prop] = prop === 'className' ? `${column[prop]} ${value}` : value
+      if (prop !== 'className' && value !== undefined) {
+        column[prop] = value
```

Comment:
This seems like different than the original expression could you please to elaborate here?